SignalServiceKit/src/Messages/Attachments/TSAttachmentStream.m

```diff
@@ -450,7 +450,7 @@ - (void)ensureThumbnail
     }
 
     if (result == nil) {
-        OWSFail(@"%@ Unable to build thumnail for attachmentId: %@", self.logTag, self.uniqueId);
```

Comment:
Why this change? Any clue what got you into this state?

Reply:
I've been testing how we handle strange media formats. There are definitely some container/codec combinations where iOS can't extract a thumbnail.  I don't think we'll be able to detect these beforehand (and treat them as a generic attachment).

Reply:
I've hit this assert a few times and it was always a result of some error in our attachment processing logic, so I would have kept it as a good sentinel. 

> I don't think we'll be able to detect these beforehand

We don't have to detect them all - this is just a debug assert. But if you think we'll be hitting this branch under normal usage in such a way that it's more distracting than the sentinel value it provides, I guess we can remove it.

Reply:
Okay, I'll put it back.